arc/utils/loops.py

```diff
@@ -189,6 +189,37 @@ def __init__(
     def _get_next_run(self) -> float:
         return self._sleep
 
+    def set_interval(
+        self,
+        *,
+        seconds: float | None = None,
+        minutes: float | None = None,
+        hours: float | None = None,
+        days: float | None = None,
+    ):
+        """Set new specified interval.
+
+        Parameters
+        ----------
+        seconds : float | None, optional
+            The number of seconds to wait before running the coroutine again.
+        minutes : float | None, optional
+            The number of minutes to wait before running the coroutine again.
+        hours : float | None, optional
+            The number of hours to wait before running the coroutine again.
+        days : float | None, optional
+            The number of days to wait before running the coroutine again.
```

Comment:
I think there should be a `!!! note` here about the fact that you need to restart the loop if you want these changes to take effect immediately. Perhaps a short 2-3 line code example too?